### DISCUSSION TITLE
How long does it take for a new repo to appear in jobs.opensafely.org?

### DISCUSSION THREAD

**Discussion by sebbacon** [opensafely/documentation#89] [Q&A] [answered]
Copied from Slack, from @rohinimathur:

> I have just created a new repo and am trying to run the analysis via the job runner. The repo isn’t showing up in the drop-down, do you know how long I have to wait till it’s picked up by the jobs.opensafely website?

**Comment by ghickman** [comment, marked as answer]
It should appear within 15 minutes of being created in the OpenSAFELY GitHub Organisation.  Don't forget to refresh the page to reload the drop down if you still can't see it.
